src/components/CreateEntry/CreateEntry.jsx

```diff
@@ -0,0 +1,23 @@
+import { useState } from 'react';
+import { createEntry } from '../../services/entries';
+
+export default function NewEntry() {
+  const [newEntry, setNewEntry] = useState('');
+
+  const handleSubmit = async () => {
+    const entry = await createEntry({ content: newEntry });
```

Comment:
When I made an entry it didn't show up on the page. Here you can do a refetch of all the entries to make sure they load after you create one.